README.md

```diff
@@ -59,9 +56,15 @@ with a **[UI]** for configuring driver related features.
 
 | Author | Discord |
 |:------:|:-------:|
-| **[Danwillm]** | `danwillm#8254` |
+| **[danwillm]** | `danwillm#8254` |
 | **[Lucas VRTech]** | `LucidVR#0001` |
 
+---
+
+**⸢ [Manual Installation] ⸥ ⸢ [Limitations] ⸥ ⸢ [Compatibility] ⸥**
```

Comment:
Looks great, but before you merge do you mind adding a link to the building doc in this line too?

Reply:
good point